src/CookiePhysicsComponent.cpp

```diff
@@ -73,11 +74,45 @@ void CookiePhysicsComponent::updatePhysics(float deltaTime) {
         // The color change has to be done here,
         // due to the bouncing off, the cookie never really hits the object.
         // TODO(nurgan) implement checking for object type and only change color if it is a "house"
-        if(objHit->getRenderComponent()->getShader().compare("Green")) {
+        if( objHit->getRenderComponent()->getShader().compare("Green")) {
             objHit->changeShader("Green");
+            cookieState.hits++;
```

Comment:
We won't want to increment the hits or score if they combo a house that has already been hit. Does this code check for that?